deployment/CP03ISSM_Deploy.csv

```diff
@@ -403,65 +403,65 @@ AR52,,AR61,,CP03ISSM-SBD11-06-METBKA000,13,1,2021-04-02T20:47:00,2021-11-11T19:3
+AR66,,,,CP03ISSM-MFD37-03-CTDBPD000,15,1,2022-04-11T21:51:00,,CGMCP-03ISSM-00015,,CGINS-CTDBPD-07239,40.367167,-70.881817,,93,94,,,
+AR66,,,,CP03ISSM-MFD37-04-DOSTAD000,15,1,2022-04-11T21:51:00,,CGMCP-03ISSM-00015,,CGINS-DOSTAD-00390,40.367167,-70.881817,,93,94,,,
+AR66,,,,CP03ISSM-MFD37-07-ZPLSCC000,15,1,2022-04-11T21:51:00,,CGMCP-03ISSM-00015,,CGINS-ZPLSCC-55074,40.367167,-70.881817,,93,94,,,
+AR66,,,,CP03ISSM-RIC21-00-CPMENG000,15,1,2022-04-11T21:51:00,,CGMCP-03ISSM-00015,,3703-00051-00001-01082,40.367167,-70.881817,,7,94,,,
+AR66,,,,CP03ISSM-RID26-00-DCLENG000,15,1,2022-04-11T21:51:00,,CGMCP-03ISSM-00015,,3718-00001-00001-00072,40.367167,-70.881817,,7,94,,,
+AR66,,,,CP03ISSM-RID26-04-VELPTA000,15,1,2022-04-11T21:51:00,,CGMCP-03ISSM-00015,,CGINS-VELPTA-12203,40.367167,-70.881817,,7,94,,,
+AR66,,,,CP03ISSM-RID26-06-PHSEND000,15,1,2022-04-11T21:51:00,,CGMCP-03ISSM-00015,,CGINS-PHSEND-P0097,40.367167,-70.881817,,7,94,,,
+AR66,,,,CP03ISSM-RID26-07-NUTNRB000,15,1,2022-04-11T21:51:00,,CGMCP-03ISSM-00015,,CGINS-NUTNRB-01063,40.367167,-70.881817,,7,94,,,
+AR66,,,,CP03ISSM-RID26-08-SPKIRB000,15,1,2022-04-11T21:51:00,,CGMCP-03ISSM-00015,,CGINS-SPKIRB-00299,40.367167,-70.881817,,7,94,,,
+AR66,,,,CP03ISSM-RID27-00-DCLENG000,15,1,2022-04-11T21:51:00,,CGMCP-03ISSM-00015,,3718-00001-00001-00008,40.367167,-70.881817,,7,94,,,
+AR66,,,,CP03ISSM-RID27-01-OPTAAD000,15,1,2022-04-11T21:51:00,,CGMCP-03ISSM-00015,,CGINS-OPTAAD-00206,40.367167,-70.881817,,7,94,,,
+AR66,,,,CP03ISSM-RID27-02-FLORTD000,15,1,2022-04-11T21:51:00,,CGMCP-03ISSM-00015,,CGINS-FLORTD-01219,40.367167,-70.881817,,7,94,,,
+AR66,,,,CP03ISSM-RID27-03-CTDBPC000,15,1,2022-04-11T21:51:00,,CGMCP-03ISSM-00015,,CGINS-CTDBPC-07208,40.367167,-70.881817,,7,94,,,
+AR66,,,,CP03ISSM-RID27-04-DOSTAD000,15,1,2022-04-11T21:51:00,,CGMCP-03ISSM-00015,,CGINS-DOSTAD-00389,40.367167,-70.881817,,7,94,,,
+AR66,,,,CP03ISSM-SBC11-00-CPMENG000,15,1,2022-04-11T21:51:00,,CGMCP-03ISSM-00015,,3703-00051-00001-00099,40.367167,-70.881817,,0,94,CPM secondary 3703-00051-00001-01079,,
+AR66,,,,CP03ISSM-SBD11-00-DCLENG000,15,1,2022-04-11T21:51:00,,CGMCP-03ISSM-00015,,3718-00001-00001-00046,40.367167,-70.881817,,0,94,,,
+AR66,,,,CP03ISSM-SBD11-01-MOPAK0000,15,1,2022-04-11T21:51:00,,CGMCP-03ISSM-00015,,CGINS-MOPAK0-43257,40.367167,-70.881817,,0,94,,,
+AR66,,,,CP03ISSM-SBD11-02-HYDGN0000,15,1,2022-04-11T21:51:00,,CGMCP-03ISSM-00015,,CGINS-HYDGN0-42316,40.367167,-70.881817,,0,94,,,
+AR66,,,,CP03ISSM-SBD11-06-METBKA000,15,1,2022-04-11T21:51:00,,CGMCP-03ISSM-00015,,CGINS-METLGR-00008,40.367167,-70.881817,,0,94,,,
+AR66,,,,CP03ISSM-SBD12-00-DCLENG000,15,1,2022-04-11T21:51:00,,CGMCP-03ISSM-00015,,3718-00001-00001-00069,40.367167,-70.881817,,0,94,,,
+AR66,,,,CP03ISSM-SBD12-03-HYDGN0000,15,1,2022-04-11T21:51:00,,CGMCP-03ISSM-00015,,CGINS-HYDGN0-41917,40.367167,-70.881817,,0,94,,,
+AR66,,,,CP03ISSM-SBD12-04-PCO2AA000,15,1,2022-04-11T21:51:00,,CGMCP-03ISSM-00015,,CGINS-PCO2AA-35260,40.367167,-70.881817,,0,94,,,
```

Comment:
deployment time/location, recovery time, and deploy/water depth updates look good. configuration previously reviewed and approved.

Reply:
All updates look good. Approved.